astropy/tests/pytest_plugins.py

```diff
@@ -43,8 +44,13 @@
 
 
 def pytest_addoption(parser):
-    parser.addoption("--remote-data", action="store_true",
+
+    # The following means that if --remote-data is not specified, the default
+    # is 'astropy', but if it is specified without arguments, it defaults to
```

Comment:
I meant the first sentence: `The following means that if --remote-data is not specified, the default is 'astropy'`.